figures/mapping-histology-labels.Rmd

```diff
@@ -184,31 +186,70 @@ display_order_df <- display_group_df %>%
                 display_order = as.numeric(display_group)) # save the factor order for text table export
 ```
 
+# Make `cancer_group_order`
+
+`cancer_group` which is a shorter form of harmonised_diagnsosis with the following edits:
+- Removed Other, Benign tumor and Dysplasia/Gliosis from cancer_group
+- Neurofibroma/Plexiform;Other updated to Neurofibroma/Plexiform
+- Non-germinomatous germ cell tumor;Teratoma updated to Teratoma
+- Anaplastic (malignant) meningioma, Meningothelial meningioma and Clear cell meningioma updated to Meningioma
+- Embryonal Tumor with Multilayered Rosettes updated to Embryonal tumor with multilayer rosettes
+
+
+Get ranks in order of big to small and make them into a new dataframe `cancer_group_order_df`. 
+ - We will remove any cancer_group with less than 5 sample counts
+ - Remove any sample which has cancer_group==NA since those are mostly Normals or non CNS tumor samples belonging 
+   to Other, Benign tumor and Dysplasia/Gliosis
+ - We will also replace "/" to ";" in cancer_group terms to work as filenames for outputs
+   
+```{r}
+cancer_group_order_df <- histology_table %>% 
+  dplyr::count(cancer_group,name = "cancer_group_n") %>% 
+  # Keep only cancer_groups with 5 or more counts
+  dplyr::filter(cancer_group_n >=5,
+                # remove NA
+		!is.na(cancer_group)) %>% 
```

Comment:
```suggestion
  # Remove NA
  dplyr::filter(!is.na(cancer_group)) %>% 
```

Reply:
We are not removing the cancer_group NA now but keeping them and adding a gray color